ras_party/controllers/pending_survey_controller.py

```diff
@@ -333,6 +356,21 @@ def get_pending_survey_by_batch_number(batch_number, session):
     return [pending_surveys.to_pending_surveys_dict() for pending_surveys in pending_surveys]
 
 
+@with_db_session
+def get_pending_survey_by_originator_respondent_id(respondent_party_id, session):
+    """
+    gets list of pending surveys against the respondent party id
+    :param respondent_party_id: respondent party id
+    :type batch_number: uuid
```

Comment:
There is no batch number param.  Also is it a UUID object or a string?  Either way, can you add it as a type hint instead of a docstring

Reply:
str will change